### ISSUE TITLE
Bug in stackblitz

### ISSUE THREAD

**Issue by Baroshem** [NozomuIkuta/h3-cors#9]
Hey hey,

It's me again :D 

One of the users of my security module (that uses your package under the hood) reported a strange issue in stackblitz. Below, I added a screenshot from my stackblitz project with security module:

![image](https://user-images.githubusercontent.com/37120330/215267656-b1d3c3c4-59d4-4f3c-8a35-0103f2e1b24e.png)

Also, the link for the stackblitz itself

https://stackblitz.com/github/baroshem/nuxt-security-stackblitz?file=package.json

The problem does not occur on local project both in dev and prod, neither in production applications so it seems that it is only caused by stackblitz. Have you encountered something like this before?

**Comment by nozomuikuta** [comment]
Hi, thank you for reporting!

To be honest, I didn't test `h3-cors` package in Stackblitz environment.

I'm not sure, but it might be the cause that `h3-cors` package peer-depends on `h3@^0.8.0`, while other Nuxt packages (e.g. `nuxt` itself) depends on `h3@^1.0.1`. But, then, I don't know why it works in other environment. 🤔 

How about yarn and pnpm? Does it result in same (i.e. doesn't work only in Stackblitz)?
If so, it seems to be Stackblitz's issue, otherwise to be npm's.

**Comment by Baroshem** [comment]
Users have started reporting this issue also on localhost with Nuxt 3.1.1

https://github.com/Baroshem/nuxt-security/issues/89

**Comment by nozomuikuta** [comment]
@Baroshem 

I found that your Nuxt module depends on `@nozomuikuta/h3-cors@^0.1.5`, which in turn depends on `h3@^0.8.6`.
Would you upgrade `@nozomuikuta/h3-cors` to the latest in your Nuxt module and check what happens?

You could test it in your local before publishing by `npm link` or the corresponding commands.

**Comment by Baroshem** [comment]
@NozomuIkuta 

Hey I can test it out, but I also did some research and it seems that the problem might be related to nuxt itself.

For the users that have been instaling my module through yarn, your package is correctly used in .nuxt folder while for those using npm, the import path is wrong (check the linked issue in my repo). I have already contacted Nuxt team to get some feedback as it seems to be nuxt related.

I will let you know once I will find something out.
